charts/fluent-bit/templates/configmap-dashboards.yaml

```diff
@@ -5,9 +5,7 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ include "fluent-bit.fullname" $ }}-dashboard-{{ trimSuffix ".json" (base $path) }}
-  {{- with $.Values.dashboards.namespace }}
-  namespace: {{ . }}
-  {{- end }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
```suggestion
  namespace: {{ default $.Release.Namespace $.Values.dashboards.namespace }}
```
The context here is inside a `range` to it needs to be escaped by a `$` and we need to continue to support the dashboard namespace being custom.